plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -232,6 +236,14 @@ public void setExcludePlugins(List<String> excludePlugins) {
         this.excludePlugins = excludePlugins;
     }
 
+    public String getAlternativePluginOrganization() {
```

Comment:
Maybe you could create as well a "public boolean isAlternative..." method to avoid having to check for null from the caller.